pages/getting-started/install-memgraph/docker.mdx

```diff
@@ -176,42 +42,14 @@ The main repositories are:
   algorithms
 - `memgraph/memgraph` includes Memgraph database and command-line interface
   [`mgconsole`](/getting-started/cli)
+- `memgraph/lab` includes a web interface [Memgraph
+Lab](/data-visualization) that helps you explore the data stored in Memgraph 
 
-## Install Memgraph using other Docker images
-
-To install Memgraph from any image, you need to adjust the `docker run` command
-in Step 2. 
-
-`memgraph/memgraph-mage` + cuGraph have a specific installation process. 
-
-### `memgraph/memgraph-platform` without MAGE
-
-Install and run the `memgraph-platform` image that doesn't include
-[MAGE](/advanced-algorithms) using the following command:
-
-```console
-docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 --name memgraph memgraph/memgraph-platform:2.10.2-memgraph2.10.1-lab2.8.1
-```
-
-Check the latest versions at [Memgraph's Docker
-Hub](https://hub.docker.com/u/memgraph).
-
-You can connect to the Memgraph instance using the command-line interface
-[mgconsole](/getting-started/cli) by running the following command in a new
-terminal:
-
-```terminal
-docker exec -it memgraph mgconsole
-```
-
-You you can also connect to Memgraph using the [Memgraph Lab web
-application](/data-visualization/user-manual) available within the
-`memgraph-platform` Docker image by visiting `localhost:3000`.
-
-To start using Memgraph in your application, use one of the [client
-libraries](/client-libraries) and follow their getting started guide.
+When you're ready to go to production you can choose
+to use the `memgraph-mage` Docker image that includes the database and the graph
+library, or just run the database using the `memgraph` image.
```

Comment:
Why this part with going to production? I would remove this part.

Reply:
Removed

Reply:
Done.